pkgs/by-name/ps/psb_status/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bash,
+  iotools,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

It seems there isn't need for a C compiler here
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```